### PR TITLE
MISO Realtime lmp

### PR DESCRIPTION
**Pull request by Thiago-NovaesB**

### CODE REVIEW

File: src/helpers/system_io_helper.jl [hunk 1]
```diff
@@ -0,0 +1,11 @@
+"""
+    _mkdir(path::AbstractString)::AbstractString
+
+Create directory if it does not exist.
+"""
+function _mkdir(path::AbstractString)::AbstractString
```
Comment (andrewrosemberg) [inline src/helpers/system_io_helper.jl]: there is already a function for this: `[mkpath`](https://docs.julialang.org/en/v1/base/file/#Base.Filesystem.mkpath)

File: src/markets/miso_market.jl [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+                continue
+            end
+            vec = [parse(Float64, x) for x in values(row[4:27])]
+            if !haskey(dict, row[1])
+                dict[row[1]] = vec
+            else
+                append!(dict[row[1]], vec)
+            end
+        end
+    end
```
Comment (andrewrosemberg) [inline src/markets/miso_market.jl:221]: Can you comment on the logic behind the control flow in this loop?

File: src/markets/miso_market.jl [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+    return dict
```
Comment (andrewrosemberg) [inline src/markets/miso_market.jl]: This should be returning a table right? If this is a temporary thing, let me know by marking the PR as [WIP].

**Review by andrewrosemberg — COMMENTED** [review]
Added a few comments, but I guess the TODOs are a sign of a WIP PR

**Comment by andrewrosemberg** [comment]
sorry closed by mistake

File: src/markets/miso_market.jl (hunk 3 above)
Comment (andrewrosemberg) [reply src/markets/miso_market.jl]: When ready add here in this PR an example of of the table with a few rows.

File: src/helpers/system_io_helper.jl (hunk 1 above)
Comment (Thiago-NovaesB) [reply src/helpers/system_io_helper.jl]: fix

Comment (Thiago-NovaesB) [reply src/helpers/system_io_helper.jl]: fix

File: src/markets/miso_market.jl (hunk 2 above)
Comment (Thiago-NovaesB) [reply src/markets/miso_market.jl:221]: done

File: src/markets/miso_market.jl (hunk 3 above)
Comment (Thiago-NovaesB) [reply src/markets/miso_market.jl]: fix
image

**Review by andrewrosemberg — APPROVED** [review]
LGTM

**Pull request merged**